nitime/timeseries.py

```diff
@@ -985,43 +981,43 @@ def __repr__(self):
     # add some methods that implement arithmetic on the timeseries data
     def __add__(self, other):
         out = self.copy()
-        out.data = out.data.__add__(other)
+        out.data = out.data.__add__(np.array(other).T)
```

Comment:
doesn't np.array make a copy by default? Wonder if it makes sense to use np.asanyarray to not make a copy of things that are already np.arrays here. Also, the transpose is surprising... Why is it necessary?